### PR TITLE
fix(route): 141JAV & 141PPV

### LINKED ISSUE #12508: error
### Routes

```routes
141jav/tag/*
```

### Full routes

```fullroutes
141jav/tag/Drama
```

### Related documentation

/141jav

### What is expected?

doesn't appear at all

### What is actually happening?

nothing happen

### Deployment information

RSSHub demo (https://rsshub.app)

### Deployment information (for self-hosted)

_No response_

### Additional info

```shell
RSSHub
Looks like something went wrong
Route requested: /141jav/tag/Drama
Error message: Response code 403 (Forbidden): target website might be blocking our access, you can host your own RSSHub instance for a better usability.
Helpful Information to provide when opening issue:
Path: /141jav/tag/Drama
Node version: v18.16.0
Git Hash: 7b533ff
        
如果您认为 RSSHub 导致了该错误，请在Github按照模版，复制本页面信息进行汇报
If you believe this is an error caused by RSSHub, please report on github
        
在线文档与支持，请访问docs.rsshub.app。

For online documentation and support please refer to docs.rsshub.app.
```

### This is not a duplicated issue

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) to ensure this bug has not already been reported

### PR DESCRIPTION
**Pull request by nczitzk**
## 该 PR 相关 Issue / Involved Issue

Close #12508

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/141jav/popular/30
/141ppv/popular/30
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [ ] 新的路由 New Route
  - [ ] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: assets/radar-rules.js [hunk 1]
```diff
@@ -286,82 +286,82 @@
...
-                        ikey = document.querySelector('div.thumbnail.is-inline > a').getAttribute('href').replace('/tag/', '').replace('/', '%2F');
-                    } else if (ikey === '' && itype === 'actress') {
-                        ikey = document.querySelector('div.card > a').getAttribute('href').replace('/actress/', '');
-                    }
-                    return `/141ppv/${itype}/${ikey}`;
-                },
-            },
-        ],
-    },
+    // '141jav.com': {
```
Comment (TonyRL) [inline assets/radar-rules.js:289]: Please remove those lines instead of commenting them.

File: lib/v2/141jav/radar.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+module.exports = {
+    '141jav.com': {
+        _name: '通用',
```
Comment (TonyRL) [inline lib/v2/141jav/radar.js:3]:
```suggestion
        _name: '141JAV',
```

File: lib/v2/141ppv/radar.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+module.exports = {
+    '141ppv.com': {
+        _name: '通用',
```
Comment (TonyRL) [inline lib/v2/141ppv/radar.js:3]:
```suggestion
        _name: '141PPV',
```

File: lib/v2/141jav/radar.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        '.': [
+            {
+                title: '今日种子',
+                docs: 'https://docs.rsshub.app/multimedia.html#141jav',
```
Comment (TonyRL) [inline lib/v2/141jav/radar.js:7]:
```suggestion
                docs: 'https://docs.rsshub.app/multimedia.html#_141jav',
```
Same as below

File: lib/v2/141ppv/radar.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        '.': [
+            {
+                title: '今日种子',
+                docs: 'https://docs.rsshub.app/multimedia.html#141pvp',
```
Comment (TonyRL) [inline lib/v2/141ppv/radar.js:7]:
```suggestion
                docs: 'https://docs.rsshub.app/multimedia.html#_141ppv',
```
Same as below

File: lib/v2/141jav/index.js [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+                .toArray()
+                .map((t) => $(t).text().trim());
+            const magnet = item.find('a[title="Magnet torrent"]').attr('href');
+            const link = item.find('a[title="Download .torrent"]').attr('href');
+            const image = item.find('.image').attr('src');
+
+            return {
+                title: `${id} ${size}`,
+                pubDate: parseDate(pubDate, 'YYYY/MM/DD'),
+                link: new URL(item.find('a').first().attr('href'), rootUrl),
```
Comment (TonyRL) [inline lib/v2/141jav/index.js:47]:
```suggestion
                link: new URL(item.find('a').first().attr('href'), rootUrl).href,
```

File: lib/v2/141ppv/index.js [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+                .toArray()
+                .map((t) => $(t).text().trim());
+            const magnet = item.find('a[title="Magnet torrent"]').attr('href');
+            const link = item.find('a[title="Download .torrent"]').attr('href');
+            const image = item.find('.image').attr('src');
+
+            return {
+                title: `${id} ${size}`,
+                pubDate: parseDate(pubDate, 'YYYY/MM/DD'),
+                link: new URL(item.find('a').first().attr('href'), rootUrl),
```
Comment (TonyRL) [inline lib/v2/141ppv/index.js:47]:
```suggestion
                link: new URL(item.find('a').first().attr('href'), rootUrl).href,
```

File: assets/radar-rules.js (hunk 1 above)
Comment (nczitzk) [reply assets/radar-rules.js:289]: 👌

**Pull request merged**